src/pages/_app.tsx

```diff
@@ -9,26 +9,49 @@ import { DefaultLayout } from '~/templates/layouts/DefaultLayout'
 import { theme } from '~/theme'
 import { trpc } from '~/utils/trpc'
 import { Provider } from 'jotai'
+import Suspense from '~/components/Suspense/Suspense'
+import ErrorBoundary from '~/components/ErrorBoundary/ErrorBoundary'
+import { Skeleton } from '@chakra-ui/react'
 
 type AppPropsWithAuthAndLayout = AppProps & {
   Component: NextPageWithLayout
 }
 
-const MyApp = (({ Component, pageProps }: AppPropsWithAuthAndLayout) => {
-  const getLayout =
-    Component.getLayout ?? ((page) => <DefaultLayout>{page}</DefaultLayout>)
-
+const MyApp = (({
+  Component,
+  pageProps,
+  ...props
+}: AppPropsWithAuthAndLayout) => {
```

Comment:
nit: no need to destructure the props, can just passthrough to `ChildWithLayout`